featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -493,7 +494,21 @@ def _calculate_transform_features(
 
             column_data = [frame[bf.get_name()] for bf in f.base_features]
 
-            feature_func = f.get_function()
+            has_series_lib_parameter = "series_library" in f.get_function.__code__.co_varnames
+
+            if isinstance(frame, dd.DataFrame):
+                if has_series_lib_parameter:
+                    feature_func = f.get_function(series_library=Library.DASK)
+                else:
+                    feature_func = f.get_function()
```

Comment:
This way users don't have to add a `series_library` parameter to their function if they don't want to